site/app/controllers/OfficeHoursQueueController.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace app\controllers;
+
+use app\libraries\Core;
+use app\libraries\response\Response;
+use app\libraries\response\WebResponse;
+use app\libraries\response\JsonResponse;
+use app\libraries\response\RedirectResponse;
+use app\models\Notification;
+use Symfony\Component\Routing\Annotation\Route;
+use app\models\OfficeHoursQueueStudent;
+use app\site\libraries\database\DatabaseQueries;
```

Comment:
`app\libraries\database\DatabaseQueries`, though doesn't look like you're actually using this (and shouldn't be really) so this can be removed.